linkerd/app/inbound/src/policy/http.rs

```diff
@@ -202,7 +202,25 @@ impl<T, N> HttpPolicyService<T, N> {
             .iter()
             .find(|a| super::is_authorized(a, self.connection.client, &self.connection.tls))
         {
-            Some(authz) => authz,
+            Some(authz) => {
+                if authz.meta.name() == "audit" {
```

Comment:
in all of these cases don't we want to check that we're dealing with a kind=default and not just an arbitrary authorization resource named audit?

Reply:
Good point, should be fixed now.